de.nagstamon.nagstamon.yml

```diff
@@ -0,0 +1,115 @@
+build-options:
+  env:
+    - BASEAPP_REMOVE_WEBENGINE=1
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --persist=.nagstamon
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess
+  - --filesystem=xdg-run/keyring
+  - --filesystem=home
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+modules:
+  - name: libsecret
+    buildsystem: meson
+    config-opts:
+      - -Dgtk_doc=false
+      - -Dintrospection=false
+      - -Dmanpage=false
+      - -Dvapi=false
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /share/man
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/libsecret/0.21/libsecret-0.21.4.tar.xz
+        sha256: 163d08d783be6d4ab9a979ceb5a4fecbc1d9660d3c34168c581301cd53912b20
+
+  - name: krb5
+    subdir: src
+    config-opts:
+      - --localstatedir=/var/lib
+      - --sbindir=${FLATPAK_DEST}/bin
+      - --disable-static
+      - --disable-rpath
+    cleanup:
+      - /bin
+      - /share/et
+      - /share/examples
+      - /share/man
+    sources:
+      - type: archive
+        url: https://github.com/krb5/krb5/archive/refs/tags/krb5-1.21.2-final.tar.gz
+        sha256: 5827fa39ead0e7f7b8a158799eb5d50b194129e81b378892caedcafe50c5c195
+      - type: shell
+        dest: src
+        commands:
+          - autoreconf -si
+
+  - name: dbus-run-session
+    buildsystem: autotools
+    cleanup: ['*']
+    sources:
+      - type: archive
+        url: https://gitlab.freedesktop.org/dbus/dbus/-/archive/dbus-1.14.10/dbus-dbus-1.14.10.tar.gz
+        sha256: 5fedcd6c17fbc62646af224e1aab11a334c463e7e7d89163a670ed24e7ab241c
+        x-checker-data:
+          type: anitya
+          project-id: 5356
+          url-template: https://gitlab.freedesktop.org/dbus/dbus/-/archive/dbus-$version/dbus-dbus-$version.tar.g00z
+
+  - name: dbus-python
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://dbus.freedesktop.org/releases/dbus-python/dbus-python-1.3.2.tar.gz
+        sha256: ad67819308618b5069537be237f8e68ca1c7fcc95ee4a121fe6845b1418248f8
+        x-checker-data:
+          type: anitya
+          project-id: 402
+          url-template: https://dbus.freedesktop.org/releases/dbus-python/dbus-python-$version.tar.gz
+
+  # external python modules
+  - python-modules.yml
+
+  - name: nagstamon
+    buildsystem: simple
+    build-commands:
+      - python3 setup.py bdist --format tar
+      - tar -xvf dist/nagstamon*.tar --strip 2 -C /app
+      - install -D nagstamon.sh /app/bin/nagstamon
+      - mkdir -p /app/share/icons/hicolor/scalable/apps/
+      - patch -i nagstamon_svg_size.patch /app/share/pixmaps/nagstamon.svg
+      - install -Dm644 /app/share/pixmaps/nagstamon.svg /app/share/icons/hicolor/scalable/apps/
+      - install -Dm644 nagstamon.appdata.xml -t /app/share/metainfo
```

Comment:
you are missing a `rename-appdata-file`

Reply:
What is rename-appdata-file used for? Is it a command? Or a manifest keyword?

Reply:
see line 10 where you use`rename-desktop-file`. work the same but for the appstream file. It's all in the documentation.